### PR TITLE
Changes data sources for Open a scene (portal item) and Feature layer selection

### PR DESCRIPTION
**Pull request by qsmmorgan**
Also cleans up the code in the "feature layer selection" sample.

### CODE REVIEW

File: arcgis-runtime-samples-macos/Features/Feature layer selection/FeatureLayerSelectionVC.swift [hunk 1]
```diff
@@ -57,28 +56,28 @@ class FeatureLayerSelectionVC: NSViewController, AGSGeoViewTouchDelegate {
...
+                                   yMax: mapPoint.y + toleranceInMapUnits,
+                                   spatialReference: map.spatialReference)
         
         //create query parameters object
         let queryParams = AGSQueryParameters()
         queryParams.geometry = envelope
         
         //select features
-        self.featureLayer.selectFeatures(withQuery: queryParams, mode: AGSSelectionMode.new) { [weak self] (queryResult:AGSFeatureQueryResult?, error:Error?) -> Void in
+        featureLayer.selectFeatures(withQuery: queryParams, mode: AGSSelectionMode.new) { [weak self] (queryResult:AGSFeatureQueryResult?, error:Error?) -> Void in
```
Comment (pgruenler) [inline arcgis-runtime-samples-macos/Features/Feature layer selection/FeatureLayerSelectionVC.swift]: Please add spaces after the colon for `queryResult` and `error`

File: arcgis-runtime-samples-macos/Features/Feature layer selection/FeatureLayerSelectionVC.swift [hunk 2]
```diff
@@ -57,28 +56,28 @@ class FeatureLayerSelectionVC: NSViewController, AGSGeoViewTouchDelegate {
     
     func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
         
-        //cancel previous query if exists
-        if let lastQuery = self.lastQuery{
-            lastQuery.cancel()
+        guard let map = map,
```
Comment (pgruenler) [inline arcgis-runtime-samples-macos/Features/Feature layer selection/FeatureLayerSelectionVC.swift]: Why are you not canceling the last query?

Comment (qsmmorgan) [reply arcgis-runtime-samples-macos/Features/Feature layer selection/FeatureLayerSelectionVC.swift]: The `lastQuery` property was never written to in the prior implementation, so I removed all the references to it.

**Comment by qsmmorgan** [comment]
I've been checking the readmes as I swap out sources and so far they've been generic toward the data. I'll hold off merging.

File: arcgis-runtime-samples-macos/Features/Feature layer selection/FeatureLayerSelectionVC.swift (hunk 2 above)
Comment (pgruenler) [reply arcgis-runtime-samples-macos/Features/Feature layer selection/FeatureLayerSelectionVC.swift]: Are you sure it was not needed rather than broken?

**Pull request merged**